frontend/src/scenes/insights/Insights.js

```diff
@@ -306,6 +307,19 @@ function _Insights() {
                                         <FunnelPeople />
                                     </Card>
                                 )}
+                            {(!allFilters.display ||
+                                allFilters.display === ACTIONS_LINE_GRAPH_LINEAR ||
+                                allFilters.display === ACTIONS_LINE_GRAPH_CUMULATIVE) &&
+                                (activeView === ViewType.TRENDS || activeView === ViewType.SESSIONS) && (
+                                    <Card>
+                                        <BindLogic
+                                            logic={trendsLogic}
+                                            props={{ dashboardItemId: null, view: activeView }}
+                                        >
+                                            <TrendLegend />
```

Comment:
I think we should only display the legend if there's something to breakdown by - either breakdown or multiple entities graphed.

Otherwise it's a waste of space most times.

Reply:
Agreed pending conclusion below (whether we show values or not)